chain/consensus/filcns/upgrades.go

```diff
@@ -1337,6 +1342,8 @@ func UpgradeActorsV8(ctx context.Context, sm *stmgr.StateManager, cache stmgr.Mi
 		return cid.Undef, xerrors.Errorf("migrating actors v7 state: %w", err)
 	}
 
+	fmt.Print(fvmLiftoffBanner)
```

Comment:
This might get printed more than once, but probably fine I guess.

(tho for future upgrades we could print them from a global `sync.Once`)